superset/assets/src/dashboard/components/Header.jsx

```diff
@@ -88,6 +100,36 @@ class Header extends React.PureComponent {
     this.props.setEditMode(!this.props.editMode);
   }
 
+  overwriteDashboard() {
+    const dashboardId = this.props.dashboardInfo.id;
+    const {
+      dashboardTitle,
+      layout: positions,
+      expandedSlices,
+      filters,
+    } = this.props;
+
+    const data = {
+      positions,
+      expanded_slices: expandedSlices,
+      dashboard_title: dashboardTitle,
+      default_filters: JSON.stringify(filters),
+    };
+
+    this.props
```

Comment:
not a blocker, but it would be better to move all of this logic to a separate action instead of overloading a component.

Reply:
fixed.